packages/session-replay-browser/src/config/types.ts

```diff
@@ -1,20 +1,33 @@
 import { Config, LogLevel, Logger } from '@amplitude/analytics-types';
 
+export interface SamplingConfig {
+  sample_rate: number;
+  capture_enabled: boolean;
+}
+
+/**
+ * NOTE(lew): defining the type like this so that it's easy to convert to
+ * the local config. This won't handle deep copies (probably), so it may
+ * need to be updated in the future.
+ */
+export type PrivacyConfig = {
```

Comment:
What about doing
```
export interface SessionReplayPrivacyConfig = {
  blockSelector?: string | string[];
};

export interface PrivacyConfig extends SessionReplayPrivacyConfig
```
(Naming could also definitely be improved here if you have any ideas)

Reply:
At this point I think I might just reuse the type as they should be equivalent.